presto-main/src/main/java/com/facebook/presto/operator/HttpPageBufferClient.java

```diff
@@ -318,8 +318,13 @@ public void onSuccess(PagesResponse result)
                         }
 
                         if (!isNullOrEmpty(taskInstanceId) && !result.getTaskInstanceId().equals(taskInstanceId)) {
-                            // TODO: update error message
-                            throw new PrestoException(REMOTE_TASK_MISMATCH, format("%s (%s)", REMOTE_TASK_MISMATCH_ERROR, fromUri(uri)));
+                            String msg = format("%s (%s).\nExpected TaskInstanceId: %s, received TaskInstanceId: %s",
```

Comment:
We normally don't put new line in error message. You can remove the `\n`.

Reply:
Removed.